pom.xml

```diff
@@ -49,6 +49,7 @@
   <modules>
     <module>js</module>
     <module>wiki</module>
+    <module>macro-pdfviewer-api</module>
```

Comment:
You could name the folder "api" to be consistent, but keep ``macro-pdfviewer-api`` as artifact ID in the POM (like the other 2 modules).